src/views/splash/l10n.json

```diff
@@ -14,6 +14,7 @@
     "messages.becomeManagerText": "{username} was promoted to manager of {studio}",
     "messages.favoriteText": "{profileLink} favorited {projectLink}",
     "messages.followText": "{profileLink} is now following {followeeLink}",
```

Comment:
Let's rename this `messages.followProfileText`, but keep the actual string the same. In transifex the translation memory should automatically translate the new string if it 100% matches an existing string.